CHANGELOG.md

```diff
@@ -5,25 +5,56 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
-## Zebra 1.X.X - XXXX-XX-XX
+## [Zebra 1.8.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.8.0) - 2024-07-01
 
-### Changed
+### Summary
 
-- We realized that a longer than `zcashd` end of support could be problematic in
-  some cases so we reverted back from 20 to 16 weeks
-  ([#8530](https://github.com/ZcashFoundation/zebra/pull/8530))
+- Zebra now drops transactions with unpaid actions in block templates and from
+  the mempool.
 - The `zebrad` binary no longer contains the scanner of shielded transactions.
   This means `zebrad` no longer contains users' viewing keys.
-- We're no longer using general conditional compilation attributes for `tor`,
-  but only feature flags instead.
-- Fixed a bug with trailing characters in the openapi spec method descriptions ([#8597](https://github.com/ZcashFoundation/zebra/pull/8597))
-- Added default constructions for several RPC method responses ([#8616](https://github.com/ZcashFoundation/zebra/pull/8616))
-- Added Windows to the list of supported platforms in Tier 2 ([#8637](https://github.com/ZcashFoundation/zebra/pull/8637))
-- Zebra now drops transactions with unpaid actions in block templates and from the mempool.
+- Support for custom Testnets and Regtest is greatly enhanced.
+- Windows is now back in the second tier of supported platforms.
+- The end-of-support time interval is set to match `zcashd`'s 16 weeks.
+- The RPC serialization of empty treestates matches `zcashd`. 
 
 ### Added
 
-- Added a page to the Zebra book describing custom Testnets ([#8636](https://github.com/ZcashFoundation/zebra/pull/8636))
+- Add an init function for a standalone `ReadStateService` ([#8595](https://github.com/ZcashFoundation/zebra/pull/8595))
+- Add new parameters for custom Testnets ([#8477](https://github.com/ZcashFoundation/zebra/pull/8477), [#8518](https://github.com/ZcashFoundation/zebra/pull/8518), [#8524](https://github.com/ZcashFoundation/zebra/pull/8524))
+- Add default constructions for several RPC method responses ([#8616](https://github.com/ZcashFoundation/zebra/pull/8616))
+- Allow custom Testnets to make peer connections and configure more parameters ([#8528](https://github.com/ZcashFoundation/zebra/pull/8528))
+- Allow configurable NU5 activation height on Regtest ([#8505](https://github.com/ZcashFoundation/zebra/pull/8505))
```

Comment:
I think this could go under the "Fixed" section too, I had just broken it in https://github.com/ZcashFoundation/zebra/pull/8475, though in this PR, #8505, we did add support for constructing Canopy proposal blocks from block templates, and a small performance improvement, if we want to mention those here as well and keep it in this section.

```suggestion
- Support constructing Canopy proposal blocks from block templates ([#8505](https://github.com/ZcashFoundation/zebra/pull/8505))
```